.nvmrc

```diff
@@ -1 +1 @@
-lts/fermium
+lts/gallium
```

Comment:
apparently npm doesn't support using lts strings for engines.node so you should just pick a dot release here. whatever is the current lts/gallium's version number would be fine

Reply:
and actually, for me MozillaBuild wants 16.17.1, so maybe that would be best.